src/VkBootstrap.h

```diff
@@ -93,11 +93,14 @@ template <typename T> class Result {
 	const T&& value () const&&   { assert (m_init); return std::move (m_value); }
 	T&&       value () &&        { assert (m_init); return std::move (m_value); }
 
+    // std::error_code associated with the error
     std::error_code error() const { assert (!m_init); return m_error.type; }
+    // optional VkResult that could of been produced due to the error
     VkResult vk_result() const { assert (!m_init); return m_error.vk_result; }
+    // Returns the internal struct that holds the errors
+    Error raw_error_struct() const { assert (!m_init); return m_error; }
```

Comment:
I don't like that it's called `raw_error_struct`. That name implies it's an extra low level thing that shouldn't generally be used - to me, at least. I'd really like to use the name `error` for this method but that's already taken. What about `full_error`?

Reply:
I say raw because its an internal type to the library, not something that is meant to be directly used.
But with that said, full_error is more descriptive than 'raw_error_struct' anyhow. I'll fix it shortly